### DISCUSSION TITLE
InvalidCertificate

### DISCUSSION THREAD

**Discussion by SebastianSosa** [quarto-dev/quarto-cli#10765] [Q&A] [answered]
Hi,

I want to install extension quarto-live but I get error 'Failed to add platform cert to root cert store: InvalidCertificate(BadEncoding)' due to a certificate issue with Deno on Windows. 

I [read]("https://github.com/quarto-dev/quarto-cli/issues/10130") that I can manually install the extension by adding the GitHub repository to the Quarto folder, but I don't know where Quarto is installed or where to place the downloaded files. 

Could you provide some guidance?

Thank you.

**Comment by mcanouil** [comment, marked as answer]
See <https://quarto.org/docs/extensions/distributing.html>.

**Reply by SebastianSosa** [reply]
Thank you very much!
